src/types/collection.ts

```diff
@@ -0,0 +1,82 @@
+import { AUTH_MODE, NODE_TYPES } from 'common/enums'
+
+export default /* GraphQL */ `
+
+  extend type Mutation {
+
+    putCollection(input: PutCollectionInput!): Collection! @auth(mode: "${AUTH_MODE.oauth}") @purgeCache(type: "${NODE_TYPES.Collection}")
+
+    deleteCollections(input: DeleteCollectionsInput!): Boolean! @auth(mode: "${AUTH_MODE.oauth}") @purgeCache(type: "${NODE_TYPES.Collection}")
```

Comment:
This won't work since the respond is a boolean not `Collection`, we should use `invalidateCache` inside mutation instead.